### PR TITLE
Enable error wildcards

### LINKED ISSUE #546: Enable analyzer build warnings as errors to be based on a wildcard
for example: `+!SqlServer.Rules.*`
or `+!SqlServer.Rules.SRN*`

### PR DESCRIPTION
**Pull request by ErikEJ**
fixes #546

### CODE REVIEW

File: test/DacpacTool.Tests/PackageAnalyzerTests.cs [hunk 1]
```diff
@@ -74,6 +74,48 @@ public void RunsAnalyzerWithWildcardSupressions()
...
+
+            // Act
+            packageAnalyzer.Analyze(result.model, result.fileInfo);
+
+            // Assert
+            testConsole.Lines.Count.ShouldBe(15);
+
+            testConsole.Lines.ShouldContain($"Analyzing package '{result.fileInfo.FullName}'");
+            testConsole.Lines.ShouldContain($"proc1.sql(1,47): Error SRD0006 : SqlServer.Rules : Avoid using SELECT *.");
+            testConsole.Lines.Count(l => l.Contains("): Error ")).ShouldBe(2);
```
Comment (cjakobsen) [inline test/DacpacTool.Tests/PackageAnalyzerTests.cs:115]: Consider adding validation of the other rule that should be returned

**Review by jmezach — APPROVED** [review]
LGTM

**Review by jeffrosenberg — APPROVED** [review]

**Comment by ErikEJ** [comment]
@jmezach Possible to get 2.7.2 out ?? 🙏

**Comment by jmezach** [comment]
@ErikEJ Release pipeline is running now

**Pull request merged**